R/data1.R

```diff
@@ -27,7 +27,7 @@
 #'     viable units for the composition data, where the former is the preferred
 #'     unit of measurement.}
 #'   \item{uncertainty}{A real value providing a measurement of uncertainty
-#'     for value. For catches and survey indices of abundance this should be
+#'     for value. For landingses and survey indices of abundance this should be
```

Comment:
typo

Reply:
I'm guessing changing from catch to landings is because discards will be modeled separately

Reply:
Yep, the issue was also that it was already landings in the data input frames and catches everywhere else which was confusing. This is a wonderful autoreplace catch with landings error :)